<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeacherPanel</class>
 <widget class="QMainWindow" name="TeacherPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(48, 62, 124);
color: white;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>50</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Teacher Panel- AMS</string>
       </property>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>121</width>
         <height>431</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);
border-radius:10px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="btn_MyStd">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>My Students</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_ClassReport">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>290</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>My Profile</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_MyRoutine">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Train System</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_StartAttendance">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>190</y>
          <width>121</width>
          <height>51</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: green;
</string>
        </property>
        <property name="text">
         <string>Start Attendance</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_teacherLogout">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>360</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 0, 0);</string>
        </property>
        <property name="text">
         <string>Log Out</string>
        </property>
       </widget>
      </widget>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>80</y>
         <width>701</width>
         <height>431</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 170, 127);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="page_1">
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>20</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>My Students</string>
         </property>
        </widget>
        <widget class="QTableWidget" name="tableWidget">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>70</y>
           <width>571</width>
           <height>271</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="lineWidth">
          <number>5</number>
         </property>
         <property name="midLineWidth">
          <number>5</number>
         </property>
         <column>
          <property name="text">
           <string>RegNo</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Full Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Level</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Semester</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Batch</string>
          </property>
         </column>
        </widget>
        <widget class="QPushButton" name="btn_viewStudent">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>352</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius:10px;
background-color: rgb(170, 170, 255);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>View Students</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2">
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>10</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>My Profile</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>210</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Full Name:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>400</x>
           <y>280</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Module:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>400</x>
           <y>210</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Teacher ID:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>280</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Password:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="txt_gname">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>240</y>
           <width>191</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border:2px solid rgb(85, 170, 127);
border-bottom-color:black;</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="txt_gmodule">
         <property name="geometry">
          <rect>
           <x>400</x>
           <y>310</y>
           <width>191</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border:2px solid rgb(85, 170, 127);
border-bottom-color:black;</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="txt_gid">
         <property name="geometry">
          <rect>
           <x>400</x>
           <y>240</y>
           <width>191</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border:2px solid rgb(85, 170, 127);
border-bottom-color:black;</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="txt_gpas">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>320</y>
           <width>191</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border:2px solid rgb(85, 170, 127);
border-bottom-color:black;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>60</y>
           <width>101</width>
           <height>81</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image: url(:/newSrc/user.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../asset/source.qrc">:/newSrc/user.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLineEdit" name="txt_searchusername">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>150</y>
           <width>191</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">border:2px solid rgb(85, 170, 127);
color: rgb(0, 0, 0);
background-color: rgb(170, 170, 255);
border-radius: 10px;
text-align:center;</string>
         </property>
         <property name="placeholderText">
          <string>Please Enter Your Username</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_searchID">
         <property name="geometry">
          <rect>
           <x>420</x>
           <y>150</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">border:2px solid black;
border-image : url(:/newSrc/loupe.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_3">
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>20</y>
           <width>131</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Train System</string>
         </property>
        </widget>
        <widget class="QProgressBar" name="progressBar">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>190</y>
           <width>421</width>
           <height>23</height>
          </rect>
         </property>
         <property name="value">
          <number>24</number>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_trainImage">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>270</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 170, 0);
color: rgb(255, 255, 255);
border-radius:10px;</string>
         </property>
         <property name="text">
          <string>Start Training</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_4">
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>10</y>
           <width>161</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Live Attendance</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_startAttendance">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>370</y>
           <width>131</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 85, 255);
color: rgb(0, 0, 0);
border-radius:10px;</string>
         </property>
         <property name="text">
          <string>Start Attendance</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>50</y>
           <width>581</width>
           <height>301</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image: url(:/newSrc/face.jpg);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../asset/source.qrc">:/newSrc/face.jpg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QPushButton" name="btn_close">
       <property name="geometry">
        <rect>
         <x>800</x>
         <y>10</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 0, 0);
border-radius:10px;</string>
       </property>
       <property name="text">
        <string>Close X</string>
       </property>
      </widget>
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>891</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 85, 127);</string>
       </property>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>0</y>
          <width>711</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Facial Recognition based Attendance System</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <zorder>widget</zorder>
      <zorder>label</zorder>
      <zorder>frame_2</zorder>
      <zorder>stackedWidget</zorder>
      <zorder>btn_close</zorder>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../asset/source.qrc"/>
 </resources>
 <connections/>
</ui>
